tests/test_self.py

```diff
@@ -991,25 +887,21 @@ def test_allow_import_of_files_found_in_modules_during_parallel_check(
 
         # For multiple jobs we could not find the `spam.py` file.
         with tmpdir.as_cwd():
-            self._runtest(
-                [
-                    "-j2",
-                    "--disable=missing-docstring, missing-final-newline",
-                    "test_directory",
-                ],
-                code=0,
-            )
+            args = [
+                "-j2",
+                "--disable=missing-docstring, missing-final-newline",
+                "test_directory",
+            ]
+            self._runtest(args, code=0)
```

Comment:
Why the additional assignment? Same for the following tests.

Reply:
Better formatting and easier comprehension of whats happenning

Reply:
Hm, I didn't think these were too problematic. In the future I would avoid such refactors, the only thing does currently does is make the test run slightly slower 😅